src/main/scala/org/constellation/concurrency/cuckoo/MutableCuckooFilter.scala

```diff
@@ -0,0 +1,67 @@
+package org.constellation.concurrency.cuckoo
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream}
+
+import cats.effect.Sync
+import net.cinnom.nanocuckoo.NanoCuckooFilter
+import org.constellation.schema.snapshot.FilterData
+import cats.syntax.all._
+
+class MutableCuckooFilter[F[_], T](filter: NanoCuckooFilter)(implicit sync: Sync[F]) {
```

Comment:
we tend to use `implicit F: Sync[F]` syntax across the project so it would be good to use it here as well